### PR TITLE
Fix MPP-2381 - Add bundle price and savings callout (DNM)

### PR DESCRIPTION
**Pull request by codemist**
This PR fixes MPP-2381
Dependent on https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/106

How to test:

Check that the homepage (banner + pricing matrix), interstitial page (banner + pricing matrix) and what's new menu contain no blank strings where bundle pricing/savings should be stated.

- [x] l10n changes have been submitted to the l10n repository, if any.
- [x] All UI revisions follow the [coding standards](https://github.com/mozilla/fx-private-relay/blob/main/docs/coding-standards.md), and use Protocol tokens where applicable (see `/frontend/src/styles/tokens.scss`).
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

**Review by Vinnl — APPROVED** [review]
image

**Comment by codemist** [comment]
Setting this as a draft for now while I wait for a solid confirmation.

File: privaterelay/settings.py [hunk 1]
```diff
@@ -724,7 +724,7 @@ def _get_initial_middleware() -> list[str]:
                 # a custom plan ID via an environment variable:
                 "id": BUNDLE_PLAN_ID_US,
                 # TODO: Insert correct price
```
[the remark this answers is not in the document]
Comment (codemist) [reply privaterelay/settings.py]: Thanks

File: frontend/pendingTranslations.ftl [hunk 2]
```diff
@@ -152,7 +152,7 @@ whatsnew-feature-bundle-snippet = Upgrade your subscription to get both { -brand
 # Variables:
 #   $monthly_price (string) - the monthly cost (including currency symbol) for a given plan. Examples: $0.99, 0,99 €
 #   $savings (string) - the percentage saved (including % symbol) for a given plan. Examples: 50%, 70%
-whatsnew-feature-bundle-body = Upgrade your subscription to get both { -brand-name-firefox-relay-premium } + { -brand-name-mozilla-vpn } for { $monthly_price }/month. Save { $savings }!
+whatsnew-feature-bundle-body = Upgrade your subscription to get both { -brand-name-firefox-relay-premium } + { -brand-name-mozilla-vpn } for { $monthly_price }/month. { $savings } off Mozilla VPN!
```
Comment (Vinnl) [inline frontend/pendingTranslations.ftl:155]:
```suggestion
whatsnew-feature-bundle-body = Upgrade your subscription to get both { -brand-name-firefox-relay-premium } + { -brand-name-mozilla-vpn } for { $monthly_price }/month. { $savings } off { -brand-name-mozilla-vpn }!
```

That said, note that this string [has already been submitted to the l10n repository](https://github.com/mozilla-l10n/fx-private-relay-l10n/blob/c6d867777f4a4ddb8998bf4c1ccc62995408c25a/en/bundle.ftl#L19), so you'll need to add a new string (also for the other changed string).

**Review by Vinnl — APPROVED** [review]

**Pull request merged**